saleor/dashboard/category/forms.py

```diff
@@ -4,24 +4,35 @@
 from django.utils.translation import pgettext_lazy
 from text_unidecode import unidecode
 
+from ...core.utils import merge_dicts
 from ...product.models import Category
+from ..seo.utils import (
+    MIN_DESCRIPTION_LENGTH, MIN_TITLE_LENGTH, SEO_HELP_TEXTS, SEO_LABELS,
+    SEO_WIDGETS)
 
 
 class CategoryForm(forms.ModelForm):
     def __init__(self, *args, **kwargs):
         self.parent_pk = kwargs.pop('parent_pk')
-        super(CategoryForm, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
+        self.fields['seo_description'].widget.attrs.update({
```

Comment:
`seo_description` and `seo_title` look like good candidates for dedicated form fields `SeoDescriptionField` and `SeoTitleField`. Those form fields could set in their constructors' proper labels, widgets and custom attributes. It would be more convenient than setting it across all the forms they're used manually. Also, this logic with merging `SEO_LABELS` etc. doesn't look very elegant.